src/vcHotkey.cpp

```diff
@@ -13,7 +13,7 @@
 namespace vcHotkey
 {
   static int target = -1;
-  static int keyBinds[vcB_Count] = {};
+  static int keyBinds[2][vcB_Count] = {};
```

Comment:
This just feels hacky, two unique variables would be much clearer too. `keyBinds` and `pendingKeyBinds` - the `bool right` used below should also change to `bool pending`, left/right terminology doesn't fit in this context.